docs/site-generator/makefile-templates/project.yml

```diff
@@ -6,3 +6,4 @@ repoVersionUrl: https://github.com/${OWNER}/${REPOSITORY}/releases/tag/${VERSION
 issuesUrl: https://github.com/${OWNER}/${REPOSITORY}/issues
 docUrl: https://godoc.org/gopkg.in/${OWNER}/${REPOSITORY}.${VERSION}
 baseURL: ${BASEURL}
+languages: ${LANGUAGES}
```

Comment:
Ouch!
I forgot to push the `aside.html` hugo template, that builds the api links using the languages